src/vma/vma_extra.h

```diff
@@ -36,6 +36,9 @@
 
 #include <stddef.h>
 #include <stdint.h>
+
+#include "utils/bullseye.h"
```

Comment:
@igor-ivanov I believe we can remove the BullseyeCoverage #pragma from this file now.
@liranoz12 added code that calls the `vma_get_api()` in commit https://github.com/Mellanox/libvma/commit/5f8ac759b0bc817691c03d45af7fddda928e58af, so in fact it is in use and can be easily covered with new test calling `vma_stats -S <fd>`